src/Stratis.Bitcoin/P2P/PeerAddress.cs

```diff
@@ -162,6 +163,33 @@ public bool Handshaked
         [JsonIgnore]
         public DateTime? LastDiscoveredFrom { get; private set; }
 
+        /// <summary>
+        /// Determine whether the peer has been banned.
+        /// <para>
+        /// This is set in <see cref="PeerBanning"/>.
+        /// </para>
+        /// </summary>
+        [JsonProperty(PropertyName = "isbanned", NullValueHandling = NullValueHandling.Ignore)]
+        public bool? IsBanned { get; set; }
+
+        /// <summary>
+        /// Reason for the banning the Peer.
+        /// <para>
+        /// This is set in <see cref="PeerBanning"/>.
+        /// </para>
+        /// </summary>
+        [JsonProperty(PropertyName = "bannedreason", NullValueHandling = NullValueHandling.Ignore)]
+        public string BannedReason { get; set; }
```

Comment:
Hmm do we need to keep the ban reason in store? this is just an arbitrary string right?

Reply:
I'll remove as stored in the log as you said.